charts/pg-db/values.yaml

```diff
@@ -11,6 +11,7 @@ finalizers:
 crVersion: 2.2.0
 image:
   repository: percona/percona-postgresql-operator
+  postgres: ""
```

Comment:
Is this a tag or something?
Maybe just call it `tag` like we have in other places.

Reply:
But this is a tag, image consists of repo:tag
The problem currently is that somewhere we have only `image` (like pgbouncer) and somewhere (like for pmm) `image.repo` and `image.tag` and in this case we have `repository` and `image` which is confusing.
So I propose to use the same in all places either just `image` which should really be in format `percona/percona-postgresql-operator:ppg15` (something like that) or `image.repo` and `image.tag` (I think this format we used in other operators).